crates/router/src/connector/dlocal/transformers.rs

```diff
@@ -0,0 +1,581 @@
+        })
+    }
+}
+
+// Type definition for Refund Response
+
+#[allow(dead_code)]
+#[derive(Debug, Serialize, Default, Deserialize, Clone)]
+#[serde(rename_all = "UPPERCASE")]
+pub enum RefundStatus {
+    SUCCESS,
+    #[default]
+    PENDING,
+    REJECTED,
+    CANCELLED,
+}
+
+impl From<RefundStatus> for enums::RefundStatus {
+    fn from(item: RefundStatus) -> Self {
+        match item {
+            RefundStatus::SUCCESS => Self::Success,
+            RefundStatus::PENDING => Self::Pending,
+            RefundStatus::REJECTED => Self::ManualReview, // Is rejected manual review?
+            RefundStatus::CANCELLED => Self::Failure,
+        }
+    }
+}
+
+//TODO: Fill the struct with respective fields
+#[derive(Default, Debug, Clone, Serialize, Deserialize)]
+pub struct RefundResponse {
+    pub id: String,
+    // pub payment_id: String,
+    // pub notification_url: String,
+    // pub amount: f64,
+    // pub currency: String,
+    pub status: RefundStatus,
+    // pub status_code: i32,
+    // pub status_detail: String,
+    // pub created_date: String,
+}
+
+impl TryFrom<types::RefundsResponseRouterData<api::Execute, RefundResponse>>
+    for types::RefundsRouterData<api::Execute>
+{
+    type Error = error_stack::Report<errors::ParsingError>;
+    fn try_from(
+        item: types::RefundsResponseRouterData<api::Execute, RefundResponse>,
+    ) -> Result<Self, Self::Error> {
+        let refund_status = enums::RefundStatus::from(item.response.status);
+        Ok(Self {
+            response: Ok(types::RefundsResponseData {
+                connector_refund_id: item.response.id,
+                refund_status,
+            }),
+            ..item.data
+        })
+    }
+}
+
+#[derive(Default, Debug, Clone, Serialize, Deserialize)]
+pub struct DlocalRefundsSyncRequest {
+    pub refund_id: String,
+}
+
+impl TryFrom<&types::RefundSyncRouterData> for DlocalRefundsSyncRequest {
+    type Error = error_stack::Report<errors::ConnectorError>;
+    fn try_from(item: &types::RefundSyncRouterData) -> Result<Self, Self::Error> {
+        let refund_id = match item.request.connector_refund_id.clone() {
+            Some(val) => val,
+            None => item.request.refund_id.clone(),
+        };
+        Ok(Self {
+            refund_id: (refund_id),
+        })
+    }
+}
+impl TryFrom<types::RefundsResponseRouterData<api::RSync, RefundResponse>>
+    for types::RefundsRouterData<api::RSync>
+{
+    type Error = error_stack::Report<errors::ParsingError>;
+    fn try_from(
+        item: types::RefundsResponseRouterData<api::RSync, RefundResponse>,
+    ) -> Result<Self, Self::Error> {
+        let refund_status = enums::RefundStatus::from(item.response.status);
+        Ok(Self {
+            response: Ok(types::RefundsResponseData {
+                connector_refund_id: item.response.id,
+                refund_status,
+            }),
+            ..item.data
+        })
+    }
+}
+
+//TODO: Fill the struct with respective fields
```

Comment:
remove this todo

Reply:
run `cargo clippy --all-features --all-targets && cargo +nightly fmt --all` and fix clippy warning